src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstanceDefinition.scala

```diff
@@ -311,6 +315,10 @@ object InstanceDefinition {
       t: PropertyType
   ): Either[DecodingFailure, InstancePropertyValue] =
     t match {
+      case PropertyType.EnumProperty(_, _) =>
```

Comment:
I guess we don't try to apply unknownValue here

Reply:
Yeah, we could do stuff like by default put it to unknown, but I am not sure how much we want to go into the depht of fdm stuff, I think keeping the support minimal while not blocking ourselves to some further integration is good for now.